src/Makefile.am

```diff
@@ -412,8 +412,6 @@ squid_SOURCES = \
 	refresh.h \
 	repl_modules.h \
 	sbuf/StringConvert.h \
-	send-announce.cc \
-	send-announce.h \
 	stat.cc \
```

Comment:
If you are removing the sender of these announcements, please remove the _recipient_ as well: src/recv-announce.cc.

Reply:
You mean that tool which was removed last year?

Reply:
Yes, my bad -- looked at the wrong branch and missed [90086c5](https://github.com/squid-cache/squid/commit/90086c5a850470d7ce5b815c2f8612c3e59d1697). Sorry about the noise.

BTW, we can mention an apparent complaints-free removal of recv-announce as an additional circumstantial evidence of the announcement feature being unused.